app/components/redactor-rte.js

```diff
@@ -61,7 +61,20 @@ export default Ember.Component.extend({
       }
     });
 
-    rte.webhookRedactor();
+    var redactorOptions = {};
+
+    // Attempt to block scripting
+    if (this.get('options.javascript') === false) {
+      redactorOptions.deniedTags = ['script'];
+      redactorOptions.allowedAttr =  [
+        ['a', ['href']],
+        ['p', 'class'],
+        ['img', ['src', 'alt']],
+        ['figure', ['data-type', 'class']]
```

Comment:
Does the table stuff need to be added here? And the code / pre tags that we allow in other rte widgets?